<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsensusGroupsDialog</class>
 <widget class="QDialog" name="ConsensusGroupsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit consensus groups</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="useDefaultGroupsRadioButton_">
     <property name="text">
      <string>Use default grouping</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="useCustomGroupsRadioButton_">
       <property name="text">
        <string>Use the following groups:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="resetToDefaultsButton_">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Reset to Defaults</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources.qrc">
         <normaloff>:/resources/images/icons/seanau/Clean.png</normaloff>:/resources/images/icons/seanau/Clean.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView_">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>22</number>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="newGroupButton_">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;New Group</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resources.qrc">
         <normaloff>:/resources/images/icons/seanau/Add.png</normaloff>:/resources/images/icons/seanau/Add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConsensusGroupsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConsensusGroupsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCustomGroupsRadioButton_</sender>
   <signal>toggled(bool)</signal>
   <receiver>newGroupButton_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCustomGroupsRadioButton_</sender>
   <signal>toggled(bool)</signal>
   <receiver>resetToDefaultsButton_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCustomGroupsRadioButton_</sender>
   <signal>toggled(bool)</signal>
   <receiver>tableView_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
